requirements.txt

```diff
@@ -7,5 +7,5 @@ rlbottraining
 # This will cause pip to auto-upgrade and stop scaring people with warning messages
 pip
 
-socketio~=0.2.1
-eventlet~=0.33.0
\ No newline at end of file
+# socketio~=0.2.1
+# eventlet~=0.33.0
```

Comment:
not sure why, but the framework doesn't like it when I add these requirements. It won't start at all, so they are there because we need them, but also they are commented because it doesn't work if we actually leave them there